code/modules/antagonists/cult/ritual.dm

```diff
@@ -24,10 +24,10 @@ This file contains the cult dagger and rune list code
 
 /obj/item/melee/cultblade/dagger/attack(mob/living/M, mob/living/user)
 	if(iscultist(M))
-		if(M.has_reagent(/datum/reagent/water/holywater)) //allows cultists to be rescued from the clutches of ordained religion
+		if(M.reagents.has_reagent(/datum/reagent/water/holywater)) //allows cultists to be rescued from the clutches of ordained religion
 			to_chat(user, "<span class='cult'>You remove the taint from [M].</span>" )
-			var/holy2unholy = M.get_reagent_amount(/datum/reagent/water/holywater)
-			M.remove_reagent(/datum/reagent/water/holywater, holy2unholy)
+			var/holy2unholy = M.reagents.get_reagent_amount(/datum/reagent/water/holywater)
+			M.reagents.del_reagent(/datum/reagent/water/holywater)
```

Comment:
will this affect holy water in a cultist's stomach?

Reply:
This is likely a case that needs more thought.
It might need to remove all in both blood and stomach.

Reply:
This should be resolved now